keyboards/tetris/tetris.h

```diff
@@ -3,16 +3,18 @@
 
 #include "quantum.h"
 
-#define LAYOUT_planck_mit( \
+#define KEYMAP( \
```

Comment:
Please revert this change.

```suggestion
#define LAYOUT_ortho_5x12( \	
```

Reply:
Should be `LAYOUT_planck_mit`.

```suggestion
#define LAYOUT_planck_mit( \
```